pkg/http/api.go

```diff
@@ -339,15 +339,15 @@ func (a *api) onOutputBindingMessage(reqCtx *fasthttp.RequestCtx) {
 func (a *api) onGetState(reqCtx *fasthttp.RequestCtx) {
 	if a.stateStores == nil || len(a.stateStores) == 0 {
 		msg := NewErrorResponse("ERR_STATE_STORE_NOT_CONFIGURED", "")
-		respondWithError(reqCtx, 400, msg)
+		respondWithError(reqCtx, 400, msg) // 404 Not Found
```

Comment:
This should remain `400`, can. you please remove the `// 404 Not Found` comment?